tox.ini

```diff
@@ -1,6 +1,34 @@
 [tox]
-envlist = py27,py33,py34,py35
+envlist = lint,py27,py33,py34,py35
 
 [testenv]
 commands = python -m pytest -s -v {posargs} tests
 deps = -r{toxinidir}/dev-requirements.txt
+
+[testenv:lint]
+skip_install = True
+deps =
+    flake8
+    flake8-bugbear
+    flake8-builtins
+    flake8-unused-arguments
+commands =
+    flake8 heudiconv
+
+[flake8]
+doctests = True
+exclude = .*/,build/,dist/,test/data,venv/
+hang-closing = False
+unused-arguments-ignore-stub-functions = True
+select = A,B,B902,C,E,E242,F,U100,W
+ignore = A003,B005,E203,E262,E266,E501,W503
```

Comment:
Nitpick (shouldn't block merge) 

IMO it's convenient to comment what these are.